Model/Transaction/Refund.php

```diff
@@ -117,10 +117,6 @@ public function build(
     public function push($forceMethod = null) {
         $refundUpdatedAt = $this->originalRefund->getUpdatedAt();
         $refundSyncedAt = $this->originalRefund->getTjSalestaxSyncDate();
-        $refundApiKey = trim($this->scopeConfig->getValue(TaxjarConfig::TAXJAR_APIKEY,
-            \Magento\Store\Model\ScopeInterface::SCOPE_STORE,
-            $this->originalOrder->getStoreId()
```

Comment:
@ryanreeves-taxjar Don't we still have to pass this store ID to get the correct API token for a refund's associated order?

Reply:
You're 100% correct.  I got carried away refactoring and forgot that we need to pass the storeId here.